### PR TITLE
Better defaults

### PR DESCRIPTION
**Pull request by benedictfischer09**
See discussion here https://github.com/doximity/es-elasticity/pull/37

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+v0.7.1
+ - Make Elasticity::Strategies::AliasIndex the default
+ - Use mapping instead of mappings, we wanna be consistent to ES not to elasticsearch-ruby
+ - Better automatic index name and document type
```
Comment (divoxx) [inline CHANGELOG]: This will prob be 0.8 since changing default strategy is a breaking change.

Comment (divoxx) [inline CHANGELOG]: We also need to add a note to the README about upgrading from one strategy to another, saying that they need to explicit define the old strategy in order to keep their indexes working.

File: README.md [hunk 2]
```diff
@@ -391,17 +391,20 @@ Note that the method takes a relation and not a class, so the following is also
 Search::User.adults.active_records(User.where(active: true))
 ```
 
-## Roadmap
+## Upgrading from 0.7.0 to 0.8.0
+The default persistance strategy changed from SingleIndex to AliasIndex in version 0.8.0 Add the follow to your Document configuration to maintain the old behavior.
```
Comment (brupm) [inline README.md]: follow to `following`

old behaviour to `legacy behaviour`.

**Comment by benedictfischer09** [comment]
@brupm I explicitly updated classes that didn't specify the behavior in https://github.com/doximity/doximity/pull/18030

**Pull request merged**